cmd/generate.go

```diff
@@ -326,6 +336,31 @@ func genUsageSnippets(cmd *cobra.Command, args []string) error {
 	return nil
 }
 
+func collectSDKGenLoggingFields(lang, schemaPath, repo, repoSubDir string) []zap.Field {
+	runLocation := os.Getenv("SPEAKEASY_RUN_LOCATION")
+	if runLocation == "" {
+		runLocation = "cli"
+	}
+	// TODO: find some way for the cli to detect version in use.
```

Comment:
Is speakeasy version passed in any env variable to the CLI. Seems like it isn't

Reply:
@ryan-timothy-albert the version is here: https://github.com/speakeasy-api/speakeasy/blob/115819e6624e5a70d7c3d18bbba81eaad0b683f4/main.go#L8

Reply:
it gets passed down to a few different commands, for example the generate gets the cli version so it can add it to its analytics and user agent

Reply:
https://github.com/speakeasy-api/speakeasy/blob/115819e6624e5a70d7c3d18bbba81eaad0b683f4/cmd/generate.go#L172

Reply:
just to clarify that version variable is set at build time, so it won't be `0.0.1`

Reply:
Perfect.